Week2/part1/app.js

```diff
@@ -0,0 +1,105 @@
+var mysql = require('mysql');
+var connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: '123123',
+  database: 'new_world',
+});
+
+connection.connect();
+
+const countryName = process.argv[2];
+
+// 1- What is the capital of country X ? (Accept X from user)
+connection.query(
+  `SELECT city.Name FROM country JOIN city ON country.code = city.CountryCode WHERE country.capital = city.ID AND country.Name = '${countryName}'`,
```

Comment:
Make sure that the user input is correctly escaped using prepared statements
https://github.com/mysqljs/mysql